wavefront-istio-adapter/values.yaml

```diff
@@ -0,0 +1,25 @@
+adapter:
+  image: vmware/wavefront-adapter-for-istio
+  tag: 0.1.3
+
+credentials:
```

Comment:
We should add the `wavefront` chart as a dependency and allow users to enable the proxy as part of this chart. See the prom storage adpater for example.

We should also add a `useProxy` property under the adapter and change this chart to:
```
adapter:
  useProxy: true
  proxyAddress: IP:PORT
```
This is within the collector for example: https://github.com/wavefrontHQ/helm/blob/df3eaa72ba97ad9b870dbf4396e2b8d63eb5993d/wavefront/values.yaml#L53

Similar to the prom storage adapter, we should support enabling the proxy within this chart. That way users can deploy the proxy/istio adapter in one go if needed.